src/posts/mitigating-pause-failures.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Pausing AI development might go wrong. How to mitigate the risks?
+description: This article addresses some of the risks of pausing AI development, and how to mitigate them.
+---
+
+We're pushing for a pause of the development of large, general AI models.
+See our [proposal](/proposal) for more details.
+
+This measure is not without risks.
+In this article, we'll address some of these risks and how to mitigate them.
+
+## Pausing too early
+
+If an AI pause happens before the risks are large enough, we might miss out on the benefits of AI.
+Ultimately, we need to balance the risks with the costs of pausing.
+
+In our view, the chance that AI will cause catastrophic risks [soon](/urgency) is already large enough to warrant a pause at this moment.
```

Comment:
As stated by Stuart Russell, when faced with an uncertain deadline, one should take the action that would be optimal given the shortest time constraint.